### PR TITLE
fix: secret key input, closes #3954

### LINKED ISSUE #3954: Seed Phrase Input Error: Firefox Browser Extension V4.34.0
When inputting the Secret Recovery Key, it's not possible to jump back on a word to correct it in case a letter is missing or need to add a missing word at the beginning or in the middle with no particular position. The cursor can be placed successfully but whenever you start to edit the letter/word, It drops at the bottom of the last word inputted and the letter/word is added there.

### PR DESCRIPTION
**Pull request by alter-eggo**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/wallet/actions/runs/5483562837).

Remove complicated secret key hide logic to simply enable blurring

### CODE REVIEW

**Comment by kyranjamie** [comment]
Can you share a video/screenshot @alter-eggo ?

File: src/app/pages/onboarding/sign-in/sign-in.tsx [hunk 1]
```diff
@@ -73,25 +70,21 @@ export function SignIn() {
...
                   onPaste={onPaste}
-                  onCopy={onCopy}
                   placeholder="Paste or type your Secret Key"
                   ref={ref as any}
                   spellCheck={false}
                   style={{ resize: 'none' }}
-                  value={
-                    isKeyMasked ? form.values.secretKey.replace(/[^ ]/g, '*') : sanitizedSecretKey
-                  }
+                  value={form.values.secretKey}
```
Comment (kyranjamie) [inline src/app/pages/onboarding/sign-in/sign-in.tsx]: Can't we remove onChange and value and bind as we could other formik inputs?

**Review by kyranjamie — APPROVED** [review]

**Review by fbwoolf — APPROVED** [review]

**Comment by alter-eggo** [comment]
https://github.com/hirosystems/wallet/assets/46521087/d3ea3609-4a6a-433a-b1f7-e8fda75889c9

File: src/app/pages/onboarding/sign-in/sign-in.tsx (hunk 1 above)
Comment (alter-eggo) [reply src/app/pages/onboarding/sign-in/sign-in.tsx]: you mean `{...form.getFieldProps('secretKey')}`?

**Comment by markmhendrickson** [comment]
I'd suggest reopening since when the user pastes a value, it shows briefly before blurred:


https://github.com/hirosystems/wallet/assets/28991/ee3895ee-ffe0-41cb-a5ff-52b2fa22241b

**Comment by alter-eggo** [comment]
@markmhx this happens bc of check for typed symbols > 0. otherwise placeholder is blurred:
Screenshot 2023-07-07 at 16 00 43

**Comment by alter-eggo** [comment]
hm, seems I found the solution, will push pr

**Pull request merged**